src/Layout/Presentation/SingleDistrict/SingleDistrict.tsx

```diff
@@ -76,41 +77,45 @@ export class SingleDistrict extends React.Component<SingleDistrictProps, {}> {
                 />
                 {calculateVulnerable && <InfoBox vulnerable={vulnerable!} vulnerableVotes={vulnerableVotes!} />}
                 <ReactTable
-                    className="-highlight -striped has-text-centered"
+                    className="-highlight -striped has-text-right"
                     data={data}
                     pageSize={data.length <= 10 ? data.length : 10}
                     showPagination={data.length > 10}
                     columns={[
                         {
-                            Header: "Parti",
+                            Header: <span className="is-pulled-left">Parti</span>,
                             accessor: "partyCode",
                             Footer: (
                                 <span>
-                                    <strong>Utvalg</strong>
+                                    <strong className="is-pulled-left">Utvalg</strong>
                                 </span>
                             ),
                             Cell: (row) => {
-                                return row.original.partyName
-                                    ? <abbr title={row.original.partyName}>{row.value}</abbr>
-                                    : row.value;
+                                return <span className="is-pulled-left">{row.original.partyName}</span>;
                             },
                         },
                         {
-                            Header: "Stemmer",
+                            Header: <span className="is-pulled-right">Stemmer</span>,
                             accessor: "votes",
                             Footer: (
                                 <span>
-                                    <strong>{data.map((value) => value.votes).reduce(toSum)}</strong>
+                                    <strong>{numberFormat(data.map((value) => value.votes).reduce(toSum))}</strong>
                                 </span>
                             ),
+                            Cell: (row) => {
+                                return numberFormat(row.value);
+                            },
                         },
                         {
-                            Header: "%",
+                            Header: <span className="is-pulled-right" >Oppslutning %</span>,
                             id: "%",
                             accessor: (d: PartyResult) => roundNumber(d.percentVotes, decimals),
+                            Cell: (row) => {
+                                return numberFormatFraction(row.value, decimals);
```

Comment:
Here you are rounding the number twice again like 151.

Reply:
Also superfluous.